sumy/summarizers/text_rank.py

```diff
@@ -20,6 +20,7 @@ class TextRankSummarizer(AbstractSummarizer):
     """
     epsilon = 1e-4
     damping = 0.85
+    delta = 1e-7
```

Comment:
Please make this one private and add comments here:

```python
# small number to prevent zero-division error, see https://github.com/miso-belica/sumy/issues/112
_delta = 1e-7
```